static/app/components/searchQueryBuilder/tokens/combobox.tsx

```diff
@@ -499,7 +499,10 @@ export function SearchQueryBuilderCombobox<
     isKeyboardDismissDisabled: true,
     shouldCloseOnBlur: true,
     shouldCloseOnInteractOutside: el => {
-      if (popoverRef.current?.contains(el)) {
+      if (
+        popoverRef.current?.contains(el) ||
+        el.closest('[data-test-id="search-query-builder"]')
```

Comment:
Usually don't want to reference `data-test-id` outside of tests, so we should use `wrapperRef` from `useSearchQueryBuilder()` instead